nbextensions/usability/toc2/README.md

```diff
@@ -16,11 +20,30 @@ The state of these two toggles is memorized and restored on reload.
 ![](image.png) 
 
 ## Configuration
-The initial configuration can be given using the IPython-contrib nbextensions facility. The maximum depth of headers to display on toc can be precised there (with a default of 4), as well as the state of the toc cell (default: false, ie not present) and the numbering of headers (true by default). The differents states and position of the floating window have reasonable defaults and can be modfied per notebook). 
-
-
+The initial configuration can be given using the IPython-contrib nbextensions facility. It includes:
+
+- The toc initial mode (floating or sidebar) 
+- The maximum depth of headers to display on toc (with a default of 6)
+- The state of the toc cell (default: false, ie not present) 
+- The numbering of headers (true by default). 
+
+The differents states and position of the floating window have reasonable defaults and can be modfied per notebook). 
+
+## Export
+It is possible to export (most of) table of contents functionalities to html. The idea is to link a relevant part of the javascript
+extension and the css, and add a small script in the html file. This is done using a template by
+```
+jupyter nbconvert FILE.ipynb --template toc
+```
+or 
+```
+jupyter nbconvert FILE.ipynb --template toc2
+```
+For the first template (toc), the files toc2.js and main.css (originally located in JUPYTER_DATA_DIR/nbextensions/usability/toc2) must reside in the same directory as intended for the html file. In the second template, these files are linked to the IPython-notebook-extensions github website. Export configuration (parameters) shall be edited directly in the template files (in templates directory JUPYTER_DATA_DIR/templates). An option "Save as HTML (with toc)" is also provided in the File menu and enable to directly convert the actual notebook. This option requires the IPython kernel and is not present with other kernels.
```

Comment:
@janschulz You just have to replace the link in the template by 
```
{% include "FILENAME" ignore missing%}
``` 
This assumes that the file is in the same directory as the notebook (full paths do not work in jinja tempates). 
I do not do it because anyway you still need an internet connection to get access to jquery.  Please make a PR with an extra template if you want